ddl/ddl_worker.go

```diff
@@ -369,32 +369,41 @@ func (d *ddl) addBatchDDLJobs2Table(tasks []*limitJobTask) error {
 		startTS = txn.StartTS()
 		return nil
 	})
-	if err == nil {
```

Comment:
This file only update two place.
One is here, make meet `err != nil` return directly.